iOSDFULibrary/Classes/Utilities/Streams/DFUStreamZip.swift

```diff
@@ -159,7 +160,7 @@ internal class DFUStreamZip : DFUStream {
                 }
                 
                 let bootloaderType = FIRMWARE_TYPE_BOOTLOADER
-                if type & bootloaderType == bootloaderType {
+                if type.rawValue & bootloaderType == bootloaderType {
```

Comment:
Instead of using raw value can't we set `FIRMWARE_TYPE_BOOTLOADER` to the equivalent enum of DFUFirmwareType ?
this is not a breaker but it would be a tiny bit cleaner

Reply:
see above